src/map/utils/petutils.cpp

```diff
@@ -727,7 +727,7 @@ namespace petutils
                    (grade::GetHPScale(grade, scaleOver30Column) * mainLevelOver30) + (grade::GetHPScale(grade, scaleOver60Column) * mainLevelOver60To75) +
                    (grade::GetHPScale(grade, scaleOver75Column) * mainLevelOver75);
 
-        // raceStat = (int32)(statScale[grade][baseValueColumn] + statScale[grade][scaleTo60Column] * (mlvl - 1));
+        // raceStat = (int32)(statScale[grade][baseValueColumn] + statScale[grade][scaleTo60Column] * (mLvl - 1));
```

Comment:
nit: commented code

Reply:
I just changed the variables to all be consistent. This has to do with automation stats which I have no idea if this is needed or not. Gonna leave it in here in case its needed for when we get around to cleaning up toau